services/eth_gateway.go

```diff
@@ -13,17 +18,70 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 )
 
-type Eth struct{}
+type Eth struct {
+	SendGas             SendGas
+	ClaimPRLs           ClaimPRLs
+	GenerateEthAddr     GenerateEthAddr
+	BuryPrl             BuryPrl
+	SendETH             SendETH
+	SendPRL             SendPRL
+	GetGasPrice         GetGasPrice
+	SubscribeToTransfer SubscribeToTransfer
+	CheckBalance        CheckBalance
+	GetCurrentBlock     GetCurrentBlock
+}
 
-// TODO: Find etherum node to connect to.
-const ethUrl = "ws://0.0.0.0"
+type SendGas func([]models.CompletedUpload) error
+type ClaimPRLs func([]models.CompletedUpload) error
+type GenerateEthAddr func() (addr string, privKey string, err error)
+type BuryPrl func()
+type SendETH func(fromAddr common.Address, toAddr common.Address, amt float64)
+type SendPRL func(fromAddr common.Address, toAddr common.Address, amt float64)
+type GetGasPrice func() (uint64, error)
+type SubscribeToTransfer func(brokerAddr common.Address, outCh chan<- types.Log)
+type CheckBalance func(common.Address)
+type GetCurrentBlock func()
 
 // Singleton client
 var (
-	client *ethclient.Client
-	mtx    sync.Mutex
+	ethUrl            string
+	MainWalletAddress common.Address
+	MainWalletKey     string
+	client            *ethclient.Client
+	mtx               sync.Mutex
+	EthWrapper        Eth
 )
 
+func init() {
+	// Load ENV variables
+	err := godotenv.Load()
+	if err != nil {
+		log.Println("Error loading .env file")
+		raven.CaptureError(err, nil)
+	}
+
+	MainWalletAddress := os.Getenv("MAIN_WALLET_ADDRESS")
+	MainWalletKey := os.Getenv("MAIN_WALLET_KEY")
+	ethUrl := os.Getenv("ETH_NODE_URL")
```

Comment:
Put all the goodies in the .env file.  The print statements below are because I was getting unused variable errors.

Reply:
Can you document all the ENV vars needed to make sure we don't miss any here:
https://docs.google.com/document/d/1B7rDoOwWV7yPgJY4FxD0DvynvjQIv4BdGOQbjIP84cI/edit

Reply:
done